### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by danfulton72** [Pyhass/Pyhiveapi#31]
**Describe the bug**
I've a working hive integration for heating controls, since the upgrade to home assistant 2021.7.0 I've been getting an error in the logs approximately every 25 seconds.

**Expected behavior**
No error in logs

**Screenshots**
Not applicable

**Additional context**
From home assistant logs

~~~
1-11-16 22:02:23.947 | ERROR    | apyhiveapi.helper.logger:error:16 - An unexpected error has occurred whilst executing getHeatOnDemand with exception <class 'KeyError'> 'autoBoost'
2021-11-16 22:02:38.315 | ERROR    | apyhiveapi.helper.logger:error:16 - An unexpected error has occurred whilst executing getHeatOnDemand with exception <class 'KeyError'> 'autoBoost'
2021-11-16 22:02:53.316 | ERROR    | apyhiveapi.helper.logger:error:16 - An unexpected error has occurred whilst executing getHeatOnDemand with exception <class 'KeyError'> 'autoBoost'
2021-11-16 22:03:08.318 | ERROR    | apyhiveapi.helper.logger:error:16 - An unexpected error has occurred whilst executing getHeatOnDemand with exception <class 'KeyError'> 'autoBoost'
2021-11-16 22:03:23.317 | ERROR    | apyhiveapi.helper.logger:error:16 - An unexpected error has occurred whilst executing getHeatOnDemand with exception <class 'KeyError'> 'autoBoost'
2021-11-16 22:03:38.317 | ERROR    | apyhiveapi.helper.logger:error:16 - An unexpected error has occurred whilst executing getHeatOnDemand with exception <class 'KeyError'> 'autoBoost'
2021-11-16 22:03:53.319 | ERROR    | apyhiveapi.helper.logger:error:16 - An unexpected error has occurred whilst executing getHeatOnDemand with exception <class 'KeyError'> 'autoBoost'
~~~

**Comment by mariotomas** [comment]
Just updated to HA 2022.6.6 and I continue to see the same errors as above 

2022-06-17 22:05:47.154 | ERROR    | apyhiveapi.helper.logger:error:16 - An unexpected error has occurred whilst executing getHeatOnDemand with exception <class 'KeyError'> 'autoBoost'
2022-06-17 22:06:02.155 | ERROR    | apyhiveapi.helper.logger:error:16 - An unexpected error has occurred whilst executing getHeatOnDemand with exception <class 'KeyError'> 'autoBoost'
2022-06-17 22:06:17.155 | ERROR    | apyhiveapi.helper.logger:error:16 - An unexpected error has occurred whilst executing getHeatOnDemand with exception <class 'KeyError'> 'autoBoost'

**Comment by KJonline** [comment]
@mariotomas This will be fixed in 2022.7.0 release in July
